### PR TITLE
Update GEOS-Chem Users page for registrations between June-Sept 2024

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard

### Describe the update

The GEOS-Chem users page has now been updated using the registered-users.xlsx dataset from AWS. This contains the registration information that users enter upon first creation of a GCClassic or GCHP run directory.

### CODE REVIEW

**Review by yantosca — CHANGES_REQUESTED** [review]
Thanks @msulprizio!  I noted a few minor issues to be resolved.  Otherwise it looks great!

File: users.html [hunk 1]
```diff
@@ -158,15 +158,21 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Atmospheric and Environmental Research (AER)
   L.marker([42.4470740448545, -71.228582651097], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Atmospheric and Environmental Research (AER)</strong><br>Lexington, Massachusetts, USA<ul><li><a href='https://www.aer.com' target='_new'>Website</a></li></ul>");
 
-  // ASU
+  // Arizona State University
   L.marker([33.423228888668525, -111.92846932676962], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Arizona State University</strong><br>Tempe, Arizona, USA<ul><li><a href='https://www.asu.edu/' target='_new'>Website</a></li></ul>");
 
+  // Asian Institute of Technology
+  L.marker([14.078499986291364, 100.61398255227519], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Asian Institute of Technology</strong><br>Pathum Thani, Thailand<ul><li><a href='https://ait.ac.th/' target='_new'>AIT</a></li><li><a href='http://www.airsompoke.com/' target='_new'</a></li>AIRSOMPOKE</ul>");
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html:165]: Yes, that is correct. That is the link the team provided.

File: users.html [hunk 2]
```diff
@@ -248,21 +260,27 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Dalhousie University
   L.marker([44.635497458, -63.58833098], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Dalhousie University</strong><br>Halifax, Nova Scotia, Canada<ul><li><a href='http://fizz.phys.dal.ca/~rachel.chang/' target='_new'>Atmospheric Processes and Composition Group</a></li></ul>");
 
-  // Delft University of Technology, Delft, Netherlands
+  // Delft University of Technology
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html:263]: Thanks. I have updated to the group page for Aircraft Noise and Climate Effects: https://www.tudelft.nl/lr/organisatie/afdelingen/control-and-operations/aircraft-noise-and-climate-effects-ance.

File: users.html [hunk 3]
```diff
@@ -521,6 +578,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // Sophia University
   L.marker([35.68306107694374, 139.73319247807564], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Sophia University</strong><br>Tokyo, Japan<ul><li><a href='https://www.seba-ken.com' target='_new'>Laboratory of Planetary Atmospheric Chemistry</a></li></ul>");
 
+  // Sorbonne University
+  L.marker([48.8483238, 2.34308173439024], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Sorbonne Université</strong><br>Paris, France<ul><li><a href='https://www.sorbonne-universite.fr/' target='_new'>Websitea></li></ul>");
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html]: Now fixed in https://github.com/geoschem/geoschem.github.io/pull/38/commits/c8adbba7c7d4f454da668e86a04a155a788814f6

File: users.html [hunk 4]
```diff
@@ -629,10 +710,13 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
...
   L.marker([55.94450373997049, -3.1889838082864825], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of Edinburgh</strong><br>Edinburgh, UK<ul><li><a href='https://sites.google.com/view/palmer-group/home' target='_new'>Atmospheric Composition Modelling Group</a></li></ul>");
 
-  // University of Exeter Exeter, UK
+  // University of Exeter
   L.marker([50.7372183244397, -3.5344501189726834], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of Exeter</strong><br>Exeter, UK<ul><li><a href='http://sites.exeter.ac.uk/atmos-chem/' target='_new'>Atmospheric Chemistry and Climate Research Group</a></li></ul>");
 
-  // University of Helsinki, Helsinki, Finland
+  // University of Graz
+  L.marker([47.07764007743541, 15.449577741625571], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of Graz</strong><br>Graz, Austria<ul><li><a href='https://physik.uni-graz.at/de/astrophysik-geophysik/' target='_new'>Department of Astrophysics and Geophysics</a></li></ul>");
+
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html:718]: Link now updated in https://github.com/geoschem/geoschem.github.io/pull/38/commits/c8adbba7c7d4f454da668e86a04a155a788814f6

File: users.html [hunk 5]
```diff
@@ -641,30 +725,36 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
...
 
-  // University of Missouri
+  // University of Missouri Kansas City
   L.marker([39.03435627435929, -94.58346697964149], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of Missouri Kansas City</strong><br>Kansas City, Missouri, USA<ul><li><a href='https://research.umkc.edu/atmoschem/' target='_new'>Christiansen Lab</a></li></ul>");
 
   // University of New South Wales Sydney, Australia
   L.marker([-33.91722230992942, 151.23155717594642], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of New South Wales</strong><br>Sydney, Australia<ul><li><a href='https://molecularphotonics.sydney' target='_new'>Molecular Photonics</a></li></ul>");
 
+  // University of Northern British Columbia
+  L.marker([53.89219072719476, -122.81330705805765], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>University of British Columbia</strong><br>Prince George, British Columbia, Canada</strong><ul><li><a href='https://www.unbc.ca/department-geography-earth-and-environmental-sciences' target='_new'>Department of Geography, Earth, and Environmental Sciences</a></li></ul>");
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html]: Thanks. Copy and paste error. Now fixed in https://github.com/geoschem/geoschem.github.io/pull/38/commits/c8adbba7c7d4f454da668e86a04a155a788814f6.

File: users.html [hunk 6]
```diff
@@ -680,17 +770,26 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
...
 
-  // Utah State University Vernal, Utah, USA
-  L.marker([40.46035402582413, -109.56532700584907], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Utah State University</strong><br>Vernal, Utah, USA<ul><li><a href='https://binghamresearch.usu.edu/index'>Bingham Research Center</a></li></ul>");
+  // Utah State University
+  L.marker([40.46035402582413, -109.56532700584907], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Utah State University</strong><br>Vernal, Utah, USA<ul><li><a href='https://binghamresearch.usu.edu/index' target='_new'>Bingham Research Center</a></li></ul>");
 
   // Utkal University
-  L.marker([20.29698158118392, 85.84044738036371], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Utkal University</strong><br>Bhubaneswar, Odisha, India<ul><li><a href='https://utkaluniversity.ac.in/faculty/dr-saroj-kumar-sahu/'>Website</a>");
+  L.marker([20.29698158118392, 85.84044738036371], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>Utkal University</strong><br>Bhubaneswar, Odisha, India<ul><li><a href='https://utkaluniversity.ac.in/faculty/dr-saroj-kumar-sahu/' target='_new'>Website</a></li></ul>");
```
[the remark this answers is not in the document]
Comment (msulprizio) [reply users.html:792]: Yes, it seems something is down on their end. Even the main university website won't load.

**Review by yantosca — APPROVED** [review]
Thanks @msulprizio.  Good to merge!

**Pull request merged**